SU-HSDO/hs_news
# adding news content type

# READY FOR REVIEW

# Summary
- Abstracted new content type

# Needed By (Date)
- soon

# Urgency
- Needed before arch

# Steps to Test

1. Enable this module
2. check for core content type fields and functionality

# Affected Projects or Products
- ARC

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-24
